clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/check_cluster_state.py

```diff
@@ -71,6 +71,7 @@ def describe_clusters():
             # The key isn't to do with clustering, skip it
             continue
 
+    exit_value = 0
```

Comment:
At the moment, I have to read to code in detail to figure out what this value means. That was fine when you called `sys.exit` inside the function, but if you return a value from the function, you need that value to be meaningful outside the context of the function. I think it means "what we want to return from `sys.exit`". That's not always going to be what users of this function want. Can you think of a more natural meaning for the return value from the function?

Whatever you do here, it would be useful to add a docstring to the function to explain what it returns.